x/lightclient/types/params.go

```diff
@@ -0,0 +1,62 @@
+package types
+
+import (
+	"time"
+
+	"github.com/cometbft/cometbft/libs/math"
+	ibctm "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint"
+	ics23 "github.com/cosmos/ics23/go"
+)
+
+var ExpectedCanonicalClientParams = ibctm.ClientState{
+	// Trust level is the fraction of the trusted validator set
```

Comment:
can you document the fact that chain id, latest height are intentionally missing